src/contexts/ConfigurationContext.tsx

```diff
@@ -396,7 +398,8 @@ export const ConfigContextProvider = ({ children }: Props) => {
       });
 
       setRerank({
-        isEnabled: isTrue(config_rerank)
+        isEnabled: isTrue(config_rerank),
+        numResults: config_rerank_num_results ?? 100,
```

Comment:
I think it will be a little more elegant to refer to the default `rerank.numResults` value, so we avoid duplicating "100":

```ts
numResults: config_rerank_num_results ?? rerank.numResults,
```

Reply:
agree